bears/python/PycodestyleBear.py

```diff
@@ -35,7 +35,7 @@ def create_arguments(
         :param max_line_length:
             Limit lines to this length.
         """
-        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s'"]
+        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s"]
```

Comment:
@jayvdb I didn't notice that... :worried: 

@sahilarora535 look at how other arguments are appended to ```arguments``` and try doing it in the same way... that will work but this is just a hacky way to go about it...  :smile: